RELEASE.md

```diff
@@ -64,6 +64,7 @@ Standby release executors perform any release outside of the normal biweekly
 schedule. The current set of standby executors are (in alphabetical order,
 see below for the rotation):
 
+- Adam Piasecki
```

Comment:
I'd be happy to promote you to the regular release rotation list (i.e. not standby).

Reply:
Thanks, sounds good to me.